autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/ScalingRequest.java

```diff
@@ -137,17 +150,14 @@ private void scaleUp(int scalingAdjustment, int hostGroupNodeCount) {
 
             LOGGER.info("Sending request to add '{}' instance(s) into host group '{}', triggered adjustmentType '{}', cluster '{}', user '{}'",
                     scalingAdjustment, hostGroup, policy.getAdjustmentType(), stackCrn, userCrn);
-            UpdateStackV4Request updateStackJson = new UpdateStackV4Request();
-            updateStackJson.setWithClusterEvent(true);
-            InstanceGroupAdjustmentV4Request instanceGroupAdjustmentJson = new InstanceGroupAdjustmentV4Request();
-            instanceGroupAdjustmentJson.setScalingAdjustment(scalingAdjustment);
-            instanceGroupAdjustmentJson.setInstanceGroup(hostGroup);
-            updateStackJson.setInstanceGroupAdjustment(instanceGroupAdjustmentJson);
-
             if (Boolean.TRUE.equals(cluster.isStopStartScalingEnabled())) {
```

Comment:
Does StopStartEnabled need to be checked again here, given the event contains the type of operation ?

Reply:
Yes, stopStartEnabled is checked when determining ScalingAdjustmentType. Removed the duplicate check